src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -2,15 +2,105 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Globalization;
+using System.Linq;
 
 namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
     public class FileWorkItemsCommand : CommandBase
     {
-        public int Run(FileWorkItemsOptions fileWorkItemsOptions)
+        private static readonly string[] s_knownFilingTargetTypes = new[] { "github", "azuredevops" };
+        private static readonly string[] s_knownFilteringStrategies = new[] { "new", "all" };
+        private static readonly string[] s_knownGroupingStrategies = new[] { "perresult" };
+
+        private FileWorkItemsOptions _options;
+
+        public int Run(FileWorkItemsOptions options)
+        {
+            if (!ValidateOptions(options)) { return 1; }
+
+            _options = options;
+
+            return 0;
+        }
+
+        private static bool ValidateOptions(FileWorkItemsOptions options)
         {
-            Console.WriteLine("The file-work-items command is not yet implemented.");
-            return 1;
+            bool valid = true;
+
+            options.FilingTargetType = options.FilingTargetType.ToLowerInvariant();
+            if (!s_knownFilingTargetTypes.Contains(options.FilingTargetType))
+            {
+                string optionDescription = CommandUtilities.GetOptionDescription<FileWorkItemsOptions>(nameof(options.FilingTargetType));
+                Console.Error.WriteLine(
+                    string.Format(
+                        CultureInfo.CurrentCulture,
```

Comment:
This is a default and this operation is not particularly sensitive. You can leave in the explicit culture, fine, but not strictly necessary, particularly if you'd like code to be more concise.

Reply:
I always supply it, to remind the reader that they should always consider which value to use.

---
In reply to: [306578696](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306578696) [](ancestors = 306578696)